build/gen_stub.php

```diff
@@ -807,33 +807,35 @@ class ArgInfo {
     public $phpDocType;
     /** @var string|null */
     public $defaultValue;
-    /** @var bool */
-    public $isSensitive;
+    /** @var AttributeInfo[] */
+    public $attributes;
 
+    /**
+     * @param AttributeInfo[] $attributes
+     */
     public function __construct(
         string $name,
         int $sendBy,
         bool $isVariadic,
         ?Type $type,
         ?Type $phpDocType,
         ?string $defaultValue,
-        bool $isSensitive
+        array $attributes
     ) {
         $this->name = $name;
         $this->sendBy = $sendBy;
         $this->isVariadic = $isVariadic;
         $this->setTypes($type, $phpDocType);
         $this->defaultValue = $defaultValue;
-        $this->isSensitive = $isSensitive;
+        $this->attributes = $attributes;
     }
 
     public function equals(ArgInfo $other): bool {
         return $this->name === $other->name
             && $this->sendBy === $other->sendBy
             && $this->isVariadic === $other->isVariadic
             && Type::equals($this->type, $other->type)
-            && $this->defaultValue === $other->defaultValue
-            && $this->isSensitive === $other->isSensitive;
+            && $this->defaultValue === $other->defaultValue;
```

Comment:
Not sure about this part. I don't believe that attributes need to be part of `equals()`, as they are not part of the `ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX` stuff.